inst/rmarkdown/templates/pdf/resources/template.tex

```diff
@@ -225,7 +225,7 @@
 $endif$
 
 $if(letterhead)$
-\usepackage[dvipsnames]{color}
+%\usepackage[dvipsnames]{color}
```

Comment:
May as well remove it.

Reply:
Done in https://github.com/eddelbuettel/linl/pull/18/commits/a833f4abdb17c85064fadb8e6344855aca039c43